### PR TITLE
removed controller routes and view related to register with email

### PR DESCRIPTION
**Pull request by andreyyy178**

### CODE REVIEW

**Review by geeksesi — CHANGES_REQUESTED** [review]
there is some Tests on : `tests/Feature/Auth` please delete them too.

**Review by geeksesi — APPROVED** [review]

File: .github/workflows/laravel.yml [hunk 1]
```diff
@@ -29,8 +29,10 @@ jobs:
                   script: |
                       cd /home/${{secrets.SSH_USERNAME}}/public_html/review-as-a-service
                       bash
-                      git pull
-                      composer i
-                      npm i
+                      GNUTLS_CPUID_OVERRIDE=0x1 git pull
+                      GNUTLS_CPUID_OVERRIDE=0x1 composer i
+                      GNUTLS_CPUID_OVERRIDE=0x1 npm i
```
Comment (vassilidev) [inline .github/workflows/laravel.yml:34]: Why

File: package.json [hunk 2]
```diff
@@ -17,5 +17,9 @@
         "react-dom": "^18.2.0",
         "tailwindcss": "^3.2.1",
         "vite": "^4.0.0"
+    },
+    "dependencies": {
+        "@fortawesome/free-brands-svg-icons": "^6.4.0",
```
Comment (vassilidev) [inline package.json:22]: Why

File: resources/js/Pages/Welcome.jsx [hunk 3]
```diff
@@ -22,12 +22,12 @@ export default function Welcome({ auth, laravelVersion, phpVersion }) {
                                 Log in
                             </Link>
 
-                            <Link
-                                href={route('register')}
-                                className="ml-4 font-semibold text-gray-600 hover:text-gray-900 dark:text-gray-400 dark:hover:text-white focus:outline focus:outline-2 focus:rounded-sm focus:outline-red-500"
-                            >
-                                Register
-                            </Link>
+                            {/*<Link*/}
```
Comment (vassilidev) [inline resources/js/Pages/Welcome.jsx:25]: Why

File: routes/auth.php [hunk 4]
```diff
@@ -19,47 +11,15 @@
...
 
-    Route::get('register', [RegisteredUserController::class, 'create'])
-                ->name('register');
-
-    Route::post('register', [RegisteredUserController::class, 'store']);
 
     Route::get('login', [AuthenticatedSessionController::class, 'create'])
                 ->name('login');
 
     Route::post('login', [AuthenticatedSessionController::class, 'store']);
```
Comment (vassilidev) [inline routes/auth.php:18]: Why

File: routes/web.php [hunk 5]
```diff
@@ -19,7 +19,7 @@
 Route::get('/', function () {
     return Inertia::render('Welcome', [
         'canLogin' => Route::has('login'),
-        'canRegister' => Route::has('register'),
+        //'canRegister' => Route::has('register'),
```
Comment (vassilidev) [inline routes/web.php:22]: Why

**Comment by vassilidev** [comment]
A lot of comment, remove the code

**Review by vassilidev — CHANGES_REQUESTED** [review]

**Comment by vassilidev** [comment]
Hello,

I'm not gonna put all the changes u need to make but don't keep code if we don't use it, don't comment it.

You miss some files and change other.

Please review it.

**Comment by vassilidev** [comment]
https://github.com/geeksesi/code-review-pals/pull/39

**Pull request closed without merging**